fix_allure.py

```diff
@@ -3,8 +3,11 @@
 
 
 openzeppelin_reports = pathlib.Path('openzeppelin-contracts/report/allure-results')
+res_files = [str(res_file) for res_file in openzeppelin_reports.glob('*-result.json')]
+print("Fix allure results: {}".format(len(res_files)))
 
-for res_file in openzeppelin_reports.glob('*-result.json'):
+
+for res_file in res_files:
```

Comment:
I prefer to use `res_file_list` because there isn't much difference between `res_file` and `res_files`.

Reply:
solved